index.html

```diff
@@ -54,7 +54,7 @@ <h2>2022.08.27(THU)~30(FRI)</h2>
             <small>@Halloway Center Sydney, Australia</small>
         </div>
     </section>
-    <section class="program-dets">
+    <section id="program-dets">
```

Comment:
This section is missing the divider right below the header and the text at the bottom.

<img width="1044" alt="Screen Shot 2022-07-27 at 9 14 48" src="https://user-images.githubusercontent.com/82426682/181269738-90ec8470-4044-4adb-9929-b5eb6570425c.png">